cert/apache_ssl/ssl-cert-snakeoil.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
+MIIC2TCCAcGgAwIBAgIUKqSP7nGboU+G9pNfr0Y6ZNxlnP0wDQYJKoZIhvcNAQEL
+BQAwFDESMBAGA1UEAwwJbG9jYWxob3N0MB4XDTIzMDUwMzE3NTYxMVoXDTMzMDQz
+MDE3NTYxMVowFDESMBAGA1UEAwwJbG9jYWxob3N0MIIBIjANBgkqhkiG9w0BAQEF
+AAOCAQ8AMIIBCgKCAQEA6DaX6O19ys1baa4wyQFRa7FofXpWJ8Ve7VgMmrgi5ww7
+Hh/DcvcF/mxsupCa8wUvlegwY6un6+r2BDqHURKPZnQoVakQ/EcKi4ZOrQp8pP86
+v4rM0IQak2R9WA3xLm/7JrIKZ84kfPTGJYZVuLnkw/mzxwVuWHt/fD0p4a482F3m
+dyk75B0JcuGM8J3xESyPeKWvE1OqU+0JqqX9B/vuE5C8g15iHqu2yK7h9nYG5nlS
+hRl54PAlXqgjoszcGQN2l2lI+g4niILGAF68Qi0qm76BrK2etdXLaCjU4jCBFl0C
+rghdP0XtAdurkGvVcTYsJ8BNl9Zab1N/pGbYSBGJOQIDAQABoyMwITAJBgNVHRME
+AjAAMBQGA1UdEQQNMAuCCWxvY2FsaG9zdDANBgkqhkiG9w0BAQsFAAOCAQEAdskV
+/sf068GIVuIJNEk74S5d2LEXdBRxqXHDOH6a4yGpimvEpMHaUNfwGFbTK6DwfIXW
+ff1Ne4aZnWoYPEOhn0GyoJXglE4cZJq84YncREwWYdgkaYwPXu11PKPzPQeyJgUV
+1CeWrLoEUwDPwP6pmxEsCZoZMDQQsYrwUZDO/74PX9iQ6s0nBKRjbMoBcmGTRl8E
+Alu12Wh3SavUr0TK2031JUuVsMhae6axth7RfCK/4dBtJ1cJyXHoDQsrUWen3Kwu
+jfmcvwINwnrF+GFl94M5yeuFT+4hPHfREM9srK2spCELUxWoD7lFPow932GbgJ3H
+O0i+tlrGQy8tlxSRiQ==
+-----END CERTIFICATE-----
```

Comment:
Not sure how this is supposed to work, but should we really push a certificate?

Reply:
It's a self-signed certificate, it is not the best to track this file but until we automated the creation process, it is more easy for developer to deploy a local environment with docker.